common/web/gesture-recognizer/src/engine/configuration/viewportZoneSource.ts

```diff
@@ -8,8 +8,8 @@ export class ViewportZoneSource implements RecognitionZoneSource {
     return DOMRect.fromRect({
       y: 0,
       x: 0,
-      height: Math.max(document.documentElement.clientWidth  || 0, window.innerWidth || 0),
-      width:  Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
+      width:  Math.max(document.documentElement.clientWidth  || 0, window.innerWidth || 0),
+      height: Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
```

Comment:
oops. Bug introduced in #6844 (https://github.com/keymanapp/keyman/pull/6844/files#diff-ff8ea78e89cd4be4698a415b52638d0b26f2bd35f6107fc017669fb712903218)